File: app/src/main/java/com/elaj/patient/models/CategoryModel.java
package com.elaj.patient.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CategoryModel {

    @SerializedName("id")
    @Expose
    public int id;
    @SerializedName("name_ar")
    @Expose
    public String nameAr;
    @SerializedName("name_en")
    @Expose
    public String nameEn;
    @SerializedName("photo")
    @Expose
    public String photo;


}

File: app/src/main/java/com/elaj/patient/models/CategoryModel.kt
package com.elaj.patient.models

import com.elaj.patient.classes.UtilityApp

class CategoryModel {
    var id: String? = null
    var name_ar: String? = null
    var name_en: String? = null
    var photo: String? = null

    val name: String?
        get() = if (UtilityApp.isEnglish) name_en else name_ar

}